.github/workflows/playgrounds_tests.yml

```diff
@@ -103,6 +108,7 @@ jobs:
       working-directory: ./apps/${{ matrix.ARGS.platform }}
       run: npm run build
 
-    # - name: Run test
-    #   working-directory: ./apps/${{ matrix.ARGS.platform }}
-    #   run: npm run test
+    - name: Run test
+      if: ${{ matrix.ARGS.platform != 'angular' }}
```

Comment:
Is it possible to comment a line in the matrix instead?
```suggestion
```

 ```
      matrix:
        ARGS: [
          # { platform: "angular" },
          { platform: "react" },
          { platform: "vue" },
        ]
```

Reply:
Unfortunately, no
Now I commented only launching of angular playground, but it still compiles, which is also important check

Reply:
👌🏻